master/reference/calicoctl/resources/profile.md

```diff
@@ -3,27 +3,29 @@ title: Profile Resource (Profile)
 canonical_url: 'https://docs.projectcalico.org/v3.4/reference/calicoctl/resources/profile'
 ---
 
-A profile resource (`Profile`) represents a set of rules which are applied
+A profile resource (`Profile`) represents a set of labels and rules which are applied
```

Comment:
Could you pull the deprecation warning into the first paragraph, maybe?  If I was skim reading, I might take the first paragraph as the message of the whole page 

How about, "Profiles provide a way to group multiple endpoints so that they inherit a shared set of labels.  For historic reasons, Profiles can also include policy rules, but that feature is deprecated in favour of (the much more flexible) NetworkPolicies and GlobalNetworkPolcies."